Moving docs from beta to v1.0 for Terms of Use API's (#11642)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/termsexpiration.md

---
title: "termsExpiration resource type"
description: "Provides additional settings when setting the scheduled expiration of the agreement."
localization_priority: Normal
ms.prod: "governance"
author: "raprakasMSFT"
doc_type: "resourcePageType"
---

# termsExpiration resource type

Namespace: microsoft.graph

> Provides additional settings when setting the scheduled expiration of the agreement.

Reviewer: ```suggestion
Provides additional settings for the scheduled expiration of the agreement.
```
